src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![crate_type="lib"]
 #![doc(html_logo_url = "https://raw.githubusercontent.com/Hoverbear/raft/master/raft.png")]
 #![doc(html_root_url = "https://hoverbear.github.io/raft/raft/")]
+#![cfg_attr(feature = "cargo-clippy", allow(doc_markdown))]
```

Comment:
Yes.  It was requesting incorrect things, and [is known to do so](https://github.com/Manishearth/rust-clippy/wiki#doc_markdown).  Specifically, it wants anything in CapsCase to be enclosed in backticks, including names that aren't code snippets, like PostgreSQL and SQLite.